cmd/src/batch_common.go

```diff
@@ -477,7 +479,13 @@ func parseBatchSpec(file string, svc *service.Service) (*batcheslib.BatchSpec, s
 		return nil, "", errors.Wrap(err, "reading batch spec")
 	}
 
-	spec, err := svc.ParseBatchSpec(data)
+	p, err := filepath.Abs(file)
```

Comment:
`file` can also be an empty string or `-`, which indicate "read from stdin". Should we in that case fall back to the working directory? Or error? Regardless, one of the two we need to do here.